website/blog/2020-05-06-jest-26.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Jest 26: Tick Tock
```

Comment:
Would recommend a more descriptive title like:

**Jest 26: Reducing install size with better defaults and tick tock release cycles**